src/arrays.js

```diff
@@ -1,61 +1,61 @@
 const getNthElement = (index, array) => {
-  // your code here
-};
-
-const arrayToCSVString = (array) => {
-  // your code here
+  if (array.length <= 4) {
+    return array[index];
+  }
+  };
+const arrayToCSVString = (array) => { 
+  return array.join()
 };
 
 const csvStringToArray = (string) => {
-  // your code here
+  return string.split(',');
 };
 
 const addToArray = (element, array) => {
-  // your code here
+  array.push(element);
 };
 
 const addToArray2 = (element, array) => {
-  // your code here
+  array.concat(element);
+  return array.concat(element);;
 };
 
 const removeNthElement = (index, array) => {
-  // your code here
+  return array.splice(index, 1);
 };
 
 const numbersToStrings = (numbers) => {
-  // your code here
+  return numbers.map(String);
 };
 
 const uppercaseWordsInArray = (strings) => {
-  // your code here
+  return (strings.map(strings => strings.toUpperCase()));
```

Comment:
These are good. It's generally better to spread this over a few lines though. It makes it easier for anybody else reading your code to see that it is a block of code.